examples/meta-node/tracer.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const {
+  diag, trace, DiagConsoleLogger, DiagLogLevel,
+} = require('@opentelemetry/api');
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+const { CollectorTraceExporter } = require('@opentelemetry/exporter-collector');
+const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+
+module.exports = () => {
+  // enable  diag to see all messages
+  diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+
+  const exporter = new CollectorTraceExporter({
+    serviceName: 'basic-service',
+  });
+
+  const provider = new NodeTracerProvider();
+  provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
+  provider.register();
+
+  registerInstrumentations({
+    instrumentations: [
+      getNodeAutoInstrumentations({
+        '@opentelemetry/instrumentation-http': {
+          applyCustomAttributesOnSpan: (span) => {
```

Comment:
do you get the typing of available options here ? I think it would be important to have them

Reply:
no, neither this is possible now with old plugin loader

Reply:
I think it could be much more easier for users if they could be typesafe, its merely just a map that gives out the type of the config for a given instrumentation name like so:
```ts
type InstrumentationConfigMap = {
  '@opentelemetry/instrumentation-http': HttpInstrumentationConfig
  etc ...
}
```

Reply:
fixed